packages/signature-controller/src/SignatureController.ts

```diff
@@ -250,6 +251,27 @@ export class SignatureController extends BaseControllerV2<
     return this.#typedMessageManager.getUnapprovedMessagesCount();
   }
 
+  /**
+   * Retrieves an array of all messages from TypedMessages and PersonalMessages.
```

Comment:
Would it be best to not mention internal properties to keep them encapsulated and keep this general?